avaframe/out3SimpPlot/outQuickPlotSimple.py

```diff
@@ -0,0 +1,119 @@
+"""
+
+This is a simple function for a quick plot of datasets and option for comparison
+between two datasets of identical shape. Also plots cross and longprofiles.
+
+This file is part of Avaframe.
+
+"""
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+from avaframe.in3Utils import fileHandlerUtils as fU
+import numpy as np
+import os
+import logging
+import shutil
+import glob
+
+# Local imports
+import avaframe.in3Utils.ascUtils as IOf
+from avaframe.out3SimpPlot.plotSettings import *
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def quickPlotSimple(avaDir, inputDir):
+    """ Plot two raster datasets of identical dimension:
+
+        Inputs:
+
+        avaDir          avalanche directory
+        suffix          result parameter abbreviation (e.g. 'ppr')
+        com1DFAOutput   folder where results to be plotted are located
+        simName         entres or null for simulation type
+        cfgR            configuration for plots
+
+        Outputs:
+
+        figure 1: plot raster data for dataset1, dataset2 and their difference
+        figure 2: plot cross and longprofiles for both datasets (ny_loc and nx_loc define location of profiles)
+        -plots are saved to Outputs/out3SimpPlot
+    """
+
+    outDir = os.path.join(avaDir, 'Outputs', 'out3SimpPlot')
+    fU.makeADir(outDir)
+
+    # Load input datasets from input directory
+    datafiles = glob.glob(inputDir+os.sep + '*.asc')
```

Comment:
Should it only work with .asc files? Maybe just take all files in the directory?

Reply:
I added now .txt files